src/app/header/show-meet-details/show-meet-details.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, Input, EventEmitter, OnInit, Output } from '@angular/core';
+import { Meet } from 'src/app/Interface/MeetInterface';
+import { AngularFireFunctions } from '@angular/fire/compat/functions';
+import { AuthService } from 'src/app/services/auth.service';
+import { ErrorHandlerService } from 'src/app/services/error-handler/error-handler.service';
+import { map, Observable } from "rxjs";
+
+
+
+
+@Component({
+  selector: 'app-show-meet-details',
+  templateUrl: './show-meet-details.component.html',
+  styleUrls: ['./show-meet-details.component.css']
+})
+export class ShowMeetDetailsComponent implements OnInit {
+
+  meetData:Meet[]=[];
+
+  @Output() getMeet: EventEmitter<string> = new EventEmitter();
```

Comment:
No need of Event emitter here, same for the Input

Reply:
Did the reqquired changes.

Reply:
did the required changes.